Source/Core/DolphinQt/MainWindow.cpp

```diff
@@ -165,6 +178,32 @@ void DMainWindow::OnOpen()
 		StartGame(filename);
 }
 
+void DMainWindow::OnBrowse()
+{
+	std::string path = ShowFolderDialog().toStdString();
+	std::vector<std::string>* iso_folder = &SConfig::GetInstance().m_ISOFolder;
+	if (!path.empty()){
```

Comment:
From the style guide:
> Use auto only in cases where it is obvious what the type being assigned is (note: 'obvious' means not having to open other files or reading the header file).

Personally I probably would have used auto here as well, but since it's not declared inside this file that that m_ISOFolder is a vector of strings, it would be violating the style guide.